src/layout/NavBar/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import './style.css';
+
+import { NavLink, useHistory } from 'react-router-dom';
+
+const NavBar = () => {
+    const history = useHistory();
+
+    return (
+        <nav>
+            <section className="page-links">
+                <NavLink exact to="/" activeClassName="current" className="navlink">Home</NavLink>
+                <NavLink to="/repopage" activeClassName="current" className="navlink">Repo Info</NavLink>
+            </section>
+            <section className="history-links">
+                <button id="back-button" onClick={history.goBack}>Go back</button>
+                <button id="forward-button" onClick={history.goForward}>Go forward</button>            
+            </section>
```

Comment:
Probably me being very picky but I feel like you could just use `div` instead of `section`

I feel like section should be used more for the main body of elements such as your search area and such.
Just checked to see if there is any convention on when to use section or not but I found this: https://html.com/tags/section/

"The <section> element is a structural HTML element used to group together related elements. Each <section> typically includes one or more heading elements and additional elements presenting related content."

Just an interesting thought. Functionally it is exactly the same so it really doesn't matter what you use 😆